components/Album/Album.test.js

```diff
@@ -0,0 +1,27 @@
+import { render, screen } from "@testing-library/react";
+import Album from "./Album";
+import userEvent from "@testing-library/user-event";
+
+const testPhotos = [
+  "/images/Carolaschlösschen1.jpg",
+  "/images/Carolaschlösschen2.jpg",
+  "/images/Carolaschlösschen3.jpg",
+];
+
+test("Image has alt text", () => {
+  render(<Album photos={testPhotos} />);
+  const photo = screen.getAllByRole("img");
+  expect(photo[0]).toHaveAttribute("alt", "a photo of the restaurant");
+});
+
+test("renders toNextPhotoButton", () => {
```

Comment:
What about the `toLastPhotoButton`?

Reply:
🥲 it is invisible, so I can't get it after render. Honestly t I have no idea, how to test them. so complex for me .

Reply:
You could mock a click on the nextPhotoButton and check whether the lastPhotoButton is now visible.